### PR TITLE
add alow-downloads to the iframe

### LINKED ISSUE #4739: Downloading is disabled in Chrome in the Preview window
## Description of the problem

We use CodeSandBox to demonstrate export in our online demos.

Steps:
Open the demo
Click the `Export` button.

Results:

Export doesn't work in Chrome because `iframe` doesn't have the allow-downloads attribute. You'll see the corresponding warning in the browser console:

>Download is disallowed. The frame initiating or instantiating the download is sandboxed, but the flag ‘allow-downloads’ is not set. See https://www.chromestatus.com/feature/5706745674465280 for more details.

## How has this issue affected you? What are you trying to accomplish?

This behavior confuses our customers. So, two questions here:
-  Is it possible to add "allow-downloads" by default ? Though it looks unsafe in a common case
- Is it possible to add an option to CLI so that we can enable  "allow-downloads" when we use your API to generate CodeSandBox in our [demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/ExportCustomMarkup/Vue/Light/).

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox   |    |
| Browser          |  Chrome 84 |
| Operating System | Windows 10 |

### PR DESCRIPTION
**Pull request by SaraVieira**
I tested in:

- Chrome: Will block any downloads done automatically
- Firefox and Brave:  Will show you a download button and not download automatically
- SaFari: will show a massive warning asking you if you wish to download that

If put on page load no website will download files automatically

Not added automatically to embedded since that is in control of the user

closes #4739

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5163/commits/ab1c8fbab79f6154542e447dfe50b22a1c215f11 is at https://pr5163.build.csb.dev/s/new.

File: packages/common/src/components/Preview/index.tsx [hunk 1]
```diff
@@ -622,8 +622,9 @@ class BasePreview extends React.PureComponent<Props, State> {
               <>
                 <StyledFrame
                   key="PREVIEW"
-                  allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
-                  sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+                  allow="accelerometer; ambient-light-sensor; 
+                   camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
```
Comment (MichaelDeBoey) [inline packages/common/src/components/Preview/index.tsx:626]:
```suggestion
                  allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
```

**Review by MichaelDeBoey — APPROVED** [review]

**Comment by mohdashraf010897** [comment]
@andrasbacsai The issue still persists.

Replicable CSB - https://ohpo6e.sse.codesandbox.io/

Click on any one of the given link-buttons that will let you save a CSV/ JSON file.
image

the console shows
image

**Comment by andrasbacsai** [comment]
I'm not working for Codesandbox.

@JamesACS, could you please check the issue? Thanks in advance.

**Comment by JamesACS** [comment]
@mohdashraf010897 - This issue is two years old. Please could you create a new one following the bug report template?

**Pull request merged**